griptape/tokenizers/base_tokenizer.py

```diff
@@ -27,19 +24,6 @@ def tokens_left(self, text: str) -> int:
         else:
             return 0
 
-    def token_count(self, text: str) -> int:
-        return len(self.encode(text))
-
-    def chunk_tokens(self, tokens: list[int]) -> Generator:
-        it = iter(tokens)
-
-        while batch := tuple(islice(it, self.max_tokens)):
-            yield batch
-
     @abstractmethod
-    def encode(self, text: str) -> list[int]:
-        ...
-
-    @abstractmethod
-    def decode(self, tokens: list[int]) -> str:
+    def token_count(self, text: str) -> int:
```

Comment:
While we're here, should this be renamed to `count_tokens`? `token_count` feels more like a property.

Reply:
Done. Also renamed `tokens_left` to `count_tokens_left`